edxval/tests/test_api.py

```diff
@@ -1335,3 +1335,30 @@ def test_update_video_image_exception(self):
                 does_not_course_id
             )
         )
+
+    def test_video_image_urls_field(self):
+        """
+        Test `VideoImage.urls` field works as expected.
+        """
+        image_urls = ['video-images/a.png', 'video-images/b.png']
+
+        # an empty list should be returned when there is no value for urls
+        self.assertEqual(self.course_video.video_image.urls, [])
+
+        # set a list with data and expect the same list to be returned
+        course_video = CourseVideo.objects.create(video=self.video, course_id='course101')
```

Comment:
Can we also assert that we get same result when access the `urls` via reverse relation.